package.json

```diff
@@ -19,11 +19,13 @@
     "@docusaurus/plugin-client-redirects": "3.4.0",
     "@docusaurus/plugin-content-docs": "3.4.0",
     "@docusaurus/preset-classic": "3.4.0",
+    "@docusaurus/theme-mermaid": "^3.4.0",
```

Comment:
I'd actually remove the `^` and pin to `3.4.0` since all docusaurus-related libraries must be the same version. Otherwise errors can happen.